### ISSUE TITLE
Tracking the Short Term Shared Storage Entropy Budget

### ISSUE THREAD

**Issue by sanjalijha** [WICG/shared-storage#165]
The long term entropy budget can be tracked in the DevTools/Application panel. Can we get support for tracking the Short Term per page load/ per origin budgets as well.

What is the best way to check these budgets in a page load session?

**Comment by jkarlin** [comment]
Displaying the budget in devtools seems reasonable to me. Will look into that.

In regards to checking the budget within a page load, we have a per-page budget per origin and a total budget for all origins. I _think_ it would be okay privacy+security wise to expose the per-page per-origin budget to a context of that origin, but the total cross-origin budget would be a cross-site leak so we couldn't expose that one. So, not sure if that's particularly useful.

**Comment by sanjalijha** [comment]
Thanks! That makes sense. Is there currently a way to check the remaining (per-page per-origin) budget of a page load? The function remainingBudget can only we called from within a worklet.

**Comment by jkarlin** [comment]
No, there is no current API to access either of the per-page budgets.

**Comment by sanjalijha** [comment]
Just to clarify, is there a command we can run currently in the DevTools Console and access the per-origin / per-page budget currently? (Before it is displayed as a UI Element)
